pkg/addons/translate.go

```diff
@@ -310,4 +310,10 @@ var Translations = map[string][]struct {
 			Template: "{{ .Config.RegistryServiceIP }}",
 		},
 	},
+	"Service/kube-service-catalog/apiserver": {
+		{
+			Path:     jsonpath.MustCompile("$.spec.externalName"),
+			Template: "servicecatalog-api.{{ .Manifest.ResourceGroup }}.svc.cluster.local",
```

Comment:
This currently supposes that HCP namespace == Manifest.ResourceGroup, which may not be the case in generality.  We would need to include the HCP namespace in the manifest.  @amanohar is it OK for us to add another field to the manifest to include the HCP namespace?

Reply:
I don't understand, why would it cause ordering problems?

Reply:
As it stands, the plugin interface rather supposes that the manifest would be passed in once only at validation time, but it seems probable that an HCP namespace would not be known at that time.  @pweil- fyi

Reply:
right, I see